frontend/package.json

```diff
@@ -33,5 +34,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
This is to not be taken in build but be installed if missing while doing a yarn install

Reply:
It would be a problem to merge this: we could deploy this site (build without `json-server`) and the `npm load:json` command would fail